app/components/mandatarissen/timeline.js

```diff
@@ -0,0 +1,45 @@
+import Component from '@glimmer/component';
+import { service } from '@ember/service';
+import moment from 'moment';
+
+export default class MandatarissenTimeline extends Component {
+  @service features;
+  @service store;
+
+  get isFeatureEnabled() {
+    return this.features.isEnabled('shacl-report');
+  }
+
+  get timelineEvents() {
+    const mandatarissen = this.args.mandatarissen;
+    const sortedMandatarissen = mandatarissen.sortBy('start');
```

Comment:
The unsorted mandatarissen are never used after sort was applied on  it
```suggestion
const sortedMandatarissen = this.args.mandatarissen.sortBy('start');
```